version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.2.0"
+version in ThisBuild := "0.2.1"
```

Comment:
Change version to `"0.3.1"`

Reply:
sure, have bumped the minor version now instead of patch version, so the new version is now `0.3.0`